src/components/modal/modal-end-chat.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import './modal.scss';
+import { useSpring, animated } from 'react-spring';
+
+
+function ModalEndChat(props){
+  const [karmaClicked, setKarmaClicked ] = React.useState({
+    karmaClicked: false
+  })
+
+  const [feedback, setFeedback] = React.useState({
+    feedback: 0
+  })
+
+  const closesQuestion = () => {
+    const token = localStorage.getItem('token');
+    const body = {
+      "karma": feedback.feedback,
+      "credits": 50,
+    }
+
+    fetch(`${process.env.REACT_APP_SERVER_URL}/questions/${props.questionId}/feedback`, {
+      method: 'PUT',
+      headers : {
+        'Authorization' : 'Bearer ' + token,
```

Comment:
DRY, you're constructing the headers every time you need to make a fetch. Even if you are not using a middleware, you still can centralize the fetch logic into a JS module, that will works as a single instance. Take advantage of it to set the headers once you log in.

The result should look like:

```js
apiClient.sendFeedback(karma, credits)
  .then(question => props.closeQuestion(question))
  …
```